src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryOperators.cs

```diff
@@ -737,23 +738,25 @@ public static Func<IRContext, FormulaValue[], FormulaValue> StringInOperator(boo
 
                 var leftStr = (StringValue)left;
                 var rightStr = (StringValue)right;
-
-                return new BooleanValue(irContext, rightStr.Value.IndexOf(leftStr.Value, exact ? StringComparison.Ordinal : StringComparison.OrdinalIgnoreCase) >= 0);
```

Comment:
Does `services` always have a CultureInfo value? In other places where I see CultureInfo being accessed (mostly in LibraryText.txt) it comes from the runner directly, not from the service provider.